src/routers/__tests__/TabRouter-test.js

```diff
@@ -139,6 +139,44 @@ describe('TabRouter', () => {
     });
   });
 
+  test('Handles passing params to nested tabs', () => {
+    const ChildTabNavigator = () => <div />;
+    ChildTabNavigator.router = TabRouter({ Boo: BareLeafRouteConfig, Bar: BareLeafRouteConfig });
+    const router = TabRouter({ Foo: BareLeafRouteConfig, Baz: { screen: ChildTabNavigator } });
+    const navAction = { type: NavigationActions.NAVIGATE, routeName: 'Baz', params: { foo: '42', bar: '43' } };
+    // const action = router.getActionForPathAndParams('Baz', { foo: '42' });
+    // expect(action).toEqual(navAction);
```

Comment:
Ok I've removed it.  It's an odd behaviour, not caused by this PR, which I wasn't sure how to go about fixing.